show/main.py

```diff
@@ -464,6 +464,72 @@ def alias(interfacename):
 
     click.echo(tabulate(body, header))
 
+@cli.group('storm-control')
+def storm_control():
+    """ show storm-control """
+    pass
+@storm_control.command('all')
+def storm_control_all():
+    """ Show storm-control """
+
+    header = ['Interface Name', 'Storm Type', 'Rate (kbps)']
+    body = []
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+
+    table = config_db.get_table('PORT_STORM_CONTROL')
+
+    #To avoid further looping below
+    if not table:
+        return
+
+    sorted_table = natsorted(table)
+
+    for storm_key in sorted_table:
+        interface_name = storm_key[0]
+        storm_type = storm_key[1]
+        #interface_name, storm_type = storm_key.split(':')
```

Comment:
How are you retreiving the names using indices(storm_key[0], storm_key[1]_ without splitting them?

Reply:
The value of storm_key is returned as follows. 
storm_key ('Ethernet0', 'broadcast')
storm_key ('Ethernet0', 'unknown-unicast')
storm_key ('Ethernet1', 'unknown-multicast')
storm_key[0] contains the interface and storm_key[1] contains the storm_type.